vmdb/config/vmdb.tmpl.yml

```diff
@@ -70,11 +70,15 @@ ems_refresh:
     get_shared_images: true
     get_public_images: false
     ignore_terminated_instances: true
-  scvmm:
+  foreman_configuration:
     :refresh_interval: 15.minutes
+  foreman_provisioning:
+    :refresh_interval: 1.hour
   full_refresh_threshold: 100
   raise_vm_snapshot_complete_if_created_within: 15.minutes
   refresh_interval: 24.hours
```

Comment:
No, they are direct properties of ems_refresh.  It looks weird in the diff because I moved scvmm section to be alphabetical.